day_1_ddo/gradient_descent.ipynb

```diff
@@ -4,39 +4,33 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "REF: https://colab.research.google.com/drive/14oFkV55gAGCjfgAbWqM4pWnroibcuo_Y?usp=sharing#scrollTo=IlT5RZ2KxQ5h"
+    "# Gradient Descent Workbook"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 76,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
-    "##############################\n",
-    "# --- Importing Packages --- #\n",
-    "##############################\n",
-    "\n",
+    "# Importing packages\n",
     "import numpy as np\n",
     "import matplotlib.pyplot as plt\n",
     "import numpy as np\n",
     "import matplotlib.pyplot as plt\n",
     "import matplotlib.pyplot as plt\n",
-    "from scipy.optimize import minimize\n"
+    "from scipy.optimize import minimize"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### **2. Gradient Descent**\n",
+    "### **1. What is Gradient Descent**\n",
     "\n",
     "Gradient descent is a simple algorithm to find the minimum value of a function. It works by iteratively \"moving downhill\" on the function's surface until you reach the lowest point.\n",
     "\n",
-    "At each step, the algorithm:\n",
-    "\n",
-    "- Calculates the slope (gradient) of the function at the current position.\n",
-    "- Updates the position by moving in the opposite direction of the slope ($ - \\nabla f(x) $ ) hence \"descent\".\n",
+    "At each step, the algorithm calculates the slope (gradient) of the function at the current position. Next, it updates the position by moving in the opposite direction of the slope $ - \\nabla f(x) $ hence \"descent\".\n",
```

Comment:
I was just thinking if it's clear that the slope is positive and that you just can say opposite direction? It doesn't really matter, but I would have phrased it as "along the slope towards the minimum"?